### ISSUE TITLE
Stop returns undefined when using useSpring with callback function

### ISSUE THREAD

**Issue by josephrexme** [react-spring/react-spring#637]
## 🐛 Bug Report

Undefined is being returned for `stop` - the third optional return, when a callback function is passed into `useSpring`

## To Reproduce

I have the following:
```js
 const [motion, setMotion, stopMotion] = useSpring(() => ({
   transform: 'scaleX(0)',
   config: { ...config.default, duration: 180000 }
 }));
```
with the jsx:
```jsx
<Track>
  <animated.div style={motion} />
</Track>
```
## Expected behavior
I expect that setMotion and stopMotion will be functions I can use to start and end my animations and indeed `setMotion` works as expected when called as:

```js
setMotion({ transform: 'scaleX(1)' });
```
However `stopMotion` keeps returning undefined.
(A clear and concise description of what you expected to happen.)

## Link to repro (highly encouraged)
Here's a reproduced demo
 [CodeSandbox demo](https://codesandbox.io/s/xv7m2yjprp)

## Environment

- `react-spring` v8.0.4
- `react` v16.8.0

**Comment by josephrexme** [comment]
I noticed from this other [demo](https://codesandbox.io/s/23w667q4kr) that in version 8.0.19 I don't get undefined but the stop still doesn't stop my animation

**Comment by aleclarson** [comment]
This is probably fixed in #632

You can check by following [this guide](https://github.com/react-spring/react-spring-examples#contributors), except do this when cloning `react-spring`:

```sh
git clone https://github.com/react-spring/react-spring -b v9
```

**Comment by aleclarson** [comment]
Fixed by #632 confirmed 🎉 

Try out v9.0.0-beta like this: `yarn add react-spring@next`

Let us know if you hit any issues here: #642

**Comment by josephrexme** [comment]
Thanks! I'll go ahead and close this now
